build.xml

```diff
@@ -7,8 +7,11 @@
 	<!-- File that contains local settings, you may need to edit this -->
 	<property file="build.properties" />
 
-	<!-- environment used during CI -->
-	<property environment="env"/>
+	<!-- Get build number from Github CI -->
+	<property environment="envgithub"/>
+	<condition property="build.number">
+		<isset property="envgithub.GITHUB_RUN_ID" />
+	</condition>
```

Comment:
I think you forgot to set the build.number value?

Reply:
I re-worked this a bit.